ci-operator/config/maistra/istio/maistra-istio-maistra-3.0.yaml

```diff
@@ -4,6 +4,18 @@ build_root:
     namespace: ci
     tag: "3.0"
   use_build_cache: true
+images:
+- dockerfile_literal: |
+    FROM quay.io/maistra-dev/maistra-builder:3.0
+    # Git checks if the user that owns the files on the filesystem match the
+    # current user.  We need to disable this check because tests in Prow are
+    # running with a random user.
+    RUN git config --system --add safe.directory '*'
+  inputs:
+    src:
+      as:
+      - quay.io/maistra-dev/maistra-builder:3.0
```

Comment:
not sure about this.
cc @yxun

Reply:
doesn't matter . We mapped the "quay.io/maistra-dev/maistra-builder:3.0" to "registry.ci.openshift.org/ci/maistra-builder:3.0" . Either one is fine.